### PR TITLE
Use a semaphore to restrict the internalization of data

### LINKED ISSUE #567: Throttle data loading
Examinations in https://github.com/equinor/ert/issues/508 shows that we should only internalize from some forward models at the time. We should support 1000 forward models with 500^3 data points.

### PR DESCRIPTION
**Pull request by ManInFez**
resolves equinor/ert#567

### CODE REVIEW

**Comment by ManInFez** [comment]
Should probably discuss the number of concurrent threads internalizing data?

File: python/res/job_queue/job_queue_node.py [hunk 1]
```diff
@@ -112,7 +113,7 @@ def _job_monitor(self, driver, max_submit):
 
             self._set_thread_status(ThreadStatus.DONE)
 
-    def run(self, driver, max_submit=2):
+    def run(self, driver, pool_sema, max_submit=2):
```
Comment (sondreso) [inline python/res/job_queue/job_queue_node.py:116]: Could we have the same order of the arguments for this function and _job_monitor?

**Review by sondreso — APPROVED** [review]
Other than my comment above and the discussion about how much we should throttle this looks good!

File: python/res/job_queue/job_queue_node.py (hunk 1 above)
Comment (ManInFez) [reply python/res/job_queue/job_queue_node.py:116]: :+1:

**Comment by markusdregi** [comment]
:+1:

**Pull request merged**